### DISCUSSION TITLE
Projects with "top up" budgets, best practice?

### DISCUSSION THREAD

**Discussion by petecooper** [kimai/kimai#4075] [Questions & Answers] [answered]
Hello.
A Customer has a Project which is ongoing with no defined end date. The Project has one or more Activities. The Project has an initial Budget to commence work, which is invoiced and prepaid. Activities are assigned to the initial Budget, which depletes the Budget. The Project shows the initial Budget still, plus depletion Activities…but the Project is ongoing, and the Budget needs to be "topped up". The Customer is sent a further invoice to add extra Budget, but it's not clear how (if at all) this can be reflected in Kimai.

What's the cleanest / sanest way to "top up" this budget with subsequent invoices from the Kimai side of things, please? Thank you.

**Comment by kevinpapst** [comment, marked as answer]
Hi Pete,
I would simply raise the budget in the project itself. And maybe leave a comment what you did exactly, so you and anyone else can later on understand what happened and why.

You could also create a follow-up project, but that would make reporting more difficult.

**Comment by petecooper** [comment]
Thank you @kevinpapst – I'll do that.
